### ISSUE TITLE
Auto-select current option in Research -> Assign

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#14]
```
Taldor, in IRC: "it would be handy if in the research options, the current
option [the one selected in the research screen listbox] is auto-selected
[in the assign listbox], instead of 'nothing'"
```

Original issue reported on code.google.com by `funnyman3595` on 29 Apr 2008 at 1:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `funnyman3595` on 29 Apr 2008 at 7:18
* Added labels: **Milestone-0.28**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git (b0b395f) and SVN (r615).

Bonus: The base dialog also auto-selects the current research task.
```

Original comment by `funnyman3595` on 1 May 2008 at 10:22
* Changed state: **Fixed**
